odoo/addons/base/views/ir_model_views.xml

```diff
@@ -347,8 +347,13 @@
                     <field name="name"/>
                     <field name="field_description"/>
                     <field name="model_id"/>
-                    <field name="ttype"/>
+                    <field name="ttype" string="Database Type"/>
+                    <field name="help"/>
                     <field name="state"/>
+                    <field name="index" string="Searchable"/>
```

Comment:
Shouldn't we name it "indexed" or something like this?

Reply:
Ok, I will.!

Reply:
Yes indeed.